src/raster/r.sun.daily/r.sun.daily.html

```diff
@@ -8,7 +8,7 @@ <h3>Output parameters explanation</h3>
 There are two basic options:
 <ul>
 <li>output series of maps (one for each day): options containing basename in their name</li>
-<li>output one map which is a sum of the intermediate maps</li>
+<li>output one map which is a sum/avg of the intermediate maps</li>
```

Comment:
You used "average" as the parameter value, but used "avg" in documentation in plain text. Use "sum or and average" here and consider using "avg" or "mean" for method parameter. Some places in GRASS GIS use avg and mean, some average. v.db.univar uses mean in the results. r.series uses average in parameters. With the new v.dissolve, you end up using avg because that's what SQLite uses. There was an effort before release of v7 to make parameters more readable by using full words, but avg is somewhat special, so I guess one I can make the argument either way.

Reply:
What about using "an aggregation of the intermediate maps" instead at this place? The aggregation methods are named under the parameter options and it wouldn't need to be inflated by a new word every time somebody adds some aggregation method.

Reply:
I took the silence as "nothing against" and changed it in 5aa2a9ace18910ef202cc0577d26dc788e73e1af.